operator/data/translateConfig/translateConfig-1.5.yaml

```diff
@@ -50,105 +50,123 @@ componentMaps:
   Base:
     ToHelmValuesTreeRoot: "global"
     HelmSubdir:           "base"
+    SkipReverseTranslate: true
   Pilot:
     ResourceType:         "Deployment"
     ResourceName:         "istio-pilot"
     ContainerName:        "discovery"
     HelmSubdir:           "istio-control/istio-discovery"
     ToHelmValuesTreeRoot: "pilot"
+    SkipReverseTranslate: false
```

Comment:
since false is the default you can omit unless true

Reply:
done